curriculum/02-data-wrangling.md

```diff
@@ -0,0 +1,91 @@
+# Data Wrangling Curriculum
+
+## Table of Contents
+
+1. [Curriculum](#1-curriculum)
+2. [Topics Not Covered](#2-topics-not-covered)
+
+## 1 Curriculum
+
+| BLU   | Name | Specialization | Status   | Content           |
+|-------|------|----------------|----------|-------------------|
+| BLU01 | Messy data | Data Wrangling | Proposed | [Content](#blu01) |
+| BLU02 | Advanced wrangling  | Data Wrangling | Proposed | [Content](#blu02) |
+| BLU03 | Data sources | Data Wrangling | Proposed | [Content](#blu03) |
+
+## Contents
+
+### BLU01
+
+#### Messy data 
+
+##### Main topics
+
+1. Alternative tabular sources 
+1. Non tabular data sources 
+1. Common problems and solutions
+
+##### Detailed curriculum
+1. beyond the csv 
+  - shuffle 
```

Comment:
should be probably: `Beyond csv` -> without `the`
first letters should be consistently capitalized or non-capitalized